src/test/java/edu/cornell/kfs/concur/ConcurUtilsTest.java

```diff
@@ -67,12 +67,12 @@ public void extractAccountNumberFromConcurValue(){
     }
     
     @Test
-    public void extracCodeFromEmptyString(){      
+    public void extractCodeFromEmptyString(){
```

Comment:
I appreciate your desire to get this unit test method corrected (and the other ones as well). However, if these methods don't have anything to do with the web service credential changes for this story, then I think these unit test fixes should be deferred to a separate user story to avoid scope creep.